data/pirate missions smuggling intro.txt

```diff
@@ -0,0 +1,1217 @@
+			choice
+				`	[Sigh] "Fine. I'll transfer it."`
+				`	"Pretty serious about that 'on-demand' line, huh? I guess it's only fair. I'll wire them to you."`
+				`	"What? No, I deserve that money."`
+					goto deserve
+			`	Alexandra chuckles. "Timely for once in your life, huh? That's great! Have a wonderful day, <first>."`
+				decline
+			
+			label deserve
+			`	Alexandra giggles, but recovers her professionalism. "Well... The offer was 500,000 for completing 5 on-demand runs. The demand was there - your runs were not. Therefore, you have until <day> to pay it back."`
+				goto meetup2
+	
+	on accept
+		set "etarip debt"
+		log `I've got until <day> to collect 500,000 credits and pay back Etarip for breaking their on-demand contract.`
+	
+	on decline
+		payment -500000
+		
+	on defer
+		"etarip payback ignored" += 1
+	
+	on fail
+		set "bounty hunters [etarip]"
+
+mission "Etarip Bounty Hunters [Pay]"
+	invisible
+	landing
+	
+	to offer
+		has "etarip debt"
+		"credits" >= 500000
+	
+	on offer
+		conversation
+			`You've collected enough money to pay back Etarip.`
+			choice
+				`	[Input Alexandra's contact number.]`
+				`	[Wait.]`
+					defer
+			`	You enter the contact number into your ship's console, and after a series of beeps, you recognize Alexandra's cheery voice.`
+			`	"Hey, <first>! Do you have the credits?"`
+			choice
+				`	"Yeah, I've got them. I'm sending them over."`
+					goto send
+				`	"I do... But I don't think I'm going to send them."`
+					goto dont
+			label send
+			`	Alexandra claps. "Yay! Glad to hear it. I'm sorry it didn't work out. I really thought it would." You put in the account number she gave you, transmit the 500,000 credits, and end the transmission.`
+				accept
+				
+			label dont
+			`	An uncomfortable pause ensues. "Well... That's unfortunate. I don't suppose there's anything I can do to convince you that's a terrible idea?"`
+			choice
+				`	"Just kidding - I'll send it over now."`
+					goto kidding
+				`	"Maybe a kiss..."`
+					goto kiss
+				`	"No, nope. I don't think so."`
+					goto no
+			label kidding
+			`	"Oh, you're a riot. I'm sorry it didn't work out. I really thought it would." You put in the account number she gave you, transmit the 500,000 credits, and end the transmission.`
+				accept
+			
+			label kiss
+			`	"Ew. You really need to work on your flirting. It's so cringey."`
+			choice
+				`	"I'm just kidding - I'll send it over now."`
+					goto kidding
+				`	"Really, though... I'm not paying it back."`
+					goto no
+			
+			label no
+			`	Alexandra sighs. "At least you're consistent." She ends the transmission.`
+				decline
+			
+	on accept
+		clear "etarip debt"
+		payment -500000
+	
+	on decline
+		set "bounty hunters [etarip]"
+		
+		
+mission "Etarip Bounty Hunters [Fail]"
+	invisible
+	landing
+
+	to offer
+		has "etarip debt"
+		has "bounty hunters [etarip]"
+		
+	npc kill
+		government Bounty
+		personality nemesis vindictive
+		fleet "Small Northern Pirates"

```

Comment:
Why not an actual Bounty Hunter fleet?

Reply:
I honestly didn't know there was a defined bounty hunter fleet. But you're right - a bounty hunter fleet does seem slightly more appropriate for bounty hunting...